app/migrations/env.py

```diff
@@ -20,7 +20,32 @@
 from flask import current_app
 config.set_main_option('sqlalchemy.url',
                        current_app.config.get('SQLALCHEMY_DATABASE_URI'))
-target_metadata = current_app.extensions['migrate'].db.metadata
+#target_metadata = current_app.extensions['migrate'].db.metadata
+from server.models import (credit_transfer, device_info, feedback, ip_address, kyc_application, models, organisation,
```

Comment:
tried various fixes found online and this is the only one that works... not great since it's so manual and repetitive, open to suggestions to make it better. 

i don't actually really understand how metadata works... the original code recognized a subset of the classes, but not only just the ones in models.py, it recognized some of the other ones in other files but not all of them, very strange. 

this is the closest to an automatic solution: https://gist.github.com/logston/e0ab3e46dd51b27f83b5d8ad359289ad
but it errors out in ways that don't make sense to me